bot.py

```diff
@@ -156,13 +157,15 @@ async def remove(ctx):
 async def inv(ctx):
     author = ctx.message.author.id
     name = ctx.message.author.name
+    set_list = [item for item in tracker.get_inv(author)]
+    one_line = "\n".join(set_list)
```

Comment:
We could let `join()` do the work instead of using list comprehension. Perhaps something like this would be better.

```suggestion
    one_line = "\n".join(tracker.get_inv(author))
```

or if we're worried about non-string items coming out of the database:

```suggestion
    one_line = "\n".join(map(lambda x: str(x), tracker.get_inv(author)))
```

Reply:
I believe I tried this and was getting each letter of the set list printing on a new line. I would definitely be willing to check again to see if this would work.